src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue

```diff
@@ -20,7 +20,8 @@
 -->
 
 <template>
-	<li class="participant-row"
+	<li :id="`${participant.source}_${participant.id}`"
```

Comment:
Is it really required to be `id` here and `anchor` in the `LeftSidebar`?

ID must be unique on a whole page, and I'm really not sure that IDs like `user_${item.id}` are guaranteed to be unique across all applications on the page. 

If it is actually only used as additional Node data to get item id from `event.target?.id` can we use some custom attr here? For example, `data-arrow-navigation-id` or at least `data-id`?

P.S. duplicated IDs is one of the most large a11y issue we currently have in Nextcloud...

Reply:
For the LeftSidebar they are used by RouterLink in `<Conversation/>` components, but not for the `<NcListItem/>` and `<Participant/>`
So I think, we could find a way to avoid it

Reply:
> For the LeftSidebar they are used by RouterLink in `<Conversation/>`

But they were not used in Talk, right?

Reply:
Router works without them, and I couldn't find anythig related or based on DOM anchor element id

They have an empty id attribute, but that's an upstream regression of NcListItem
![image](https://github.com/nextcloud/spreed/assets/93392545/7b3cef0e-ed0f-4568-9fea-6e5000034ea0)